Umbraco CI/CD Flow - first take (#5476)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/umbraco-cicd/KnownLimitationsAndConsiderations.md

# Known Limitations and Considerations

## Potential Limitations and Considerations

_Format Restrictions_
- The packaged artifact from your CI/CD pipeline must adhere to the Umbraco Cloud API's required format, which is a zip source file. This could necessitate changes to your existing build and packaging scripts.

_Back Office Limitations_

> - To ensure smooth execution of the Umbraco CI/CD Flow, it is highly recommended to adhere to a specific principle: refrain from making any alterations to the schema, except in the leftmost environment within the Umbraco Cloud back offices. The intention behind this principle is to prevent conflicts that could potentially arise due to simultaneous modifications made in various environments.

Reviewer: ```suggestion
- To ensure smooth execution of the Umbraco CI/CD Flow, it is highly recommended to adhere to a specific principle: refrain from making any alterations to the schema, except in the leftmost environment within the Umbraco Cloud back offices. The intention behind this principle is to prevent conflicts that could potentially arise due to simultaneous modifications made in different environments.
```

_Additional Build Step_
- The flow feature adds an extra build to the deployment process. As a result, it takes longer to post to Umbraco Cloud using Umbraco CI/CD Flow compared to standard deployments.

_Conflict Management_
- Given the necessity to avoid changes in other environments, the lack of strict coordination among multiple teams or individuals working on the same project elevates the risk of conflicts.

> As we continue to prototype and gather insights from beta users, there are several known limitations and considerations to be aware of:

Reviewer: ```suggestion
As we continue to prototype and gather insights from beta users, there are a number of known limitations and considerations to be aware of:
```

> As we continue to prototype and gather insights from beta users, there are a number of known limitations and considerations to be aware of:

Reviewer 2: ```suggestion
As we continue to prototype and gather insights from beta users, there are some known limitations and considerations to be aware of:
```

> ## Key Points to Consider:

Reviewer: ```suggestion
## Key Points to Consider
```
